__tests__/app.test.js

```diff
@@ -138,15 +138,146 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/100000")
       .expect(404)
       .then(({ body }) => {
-        expect(body.message).toBe("article not found");
+        expect(body.message).toBe("Article not found");
       });
   });
   it("responds with status 400 for invalid non numeric id", () => {
     return request(app)
       .get("/api/articles/banana")
       .expect(400)
       .then(({ body }) => {
-        expect(body.msg).toBe("invalid id");
+        expect(body.message).toBe("Invalid ID");
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
+  it("responds with comments array and status 200", () => {
+    return request(app)
+      .get("/api/articles/6/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeInstanceOf(Array);
+      });
+  });
+  it("responds with all comments, if there are comments", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toBe(11);
+      });
+  });
+  it("responds with status 200 and message for no comments", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.message).toBe("No comments");
+        expect(body.comments.length).toBe(0);
+      });
+  });

```

Comment:
I do like that you have all of these tests in here, but i would recommend combining them so you dont have many seperate requests being sent which are testing for very similar things.